ban/tests/http/test_position.py

```diff
@@ -442,3 +442,45 @@ def test_get_position_collection_filtered_by_bbox_is_paginated(get, url):
     assert 'previous' in page2
     resp = get(page2['previous'])
     assert resp.json == page1
+
+
+@authorize
+def test_get_position_collection_filtered_by_1_kind_param(get, url):
+    PositionFactory(kind='entrance')
+    PositionFactory(kind='exit')
+    resp = get(url('position', query_string={'kind': 'entrance'}))
+    assert resp.status == falcon.HTTP_200
+    assert resp.json['total'] == 1
```

Comment:
Let's also make sure the result is the one we expect (needs to be done for test_postcode too I guess).